x-pack/plugins/cases/server/telemetry/queries/cases.ts

```diff
@@ -68,28 +70,55 @@ export const getCasesTelemetryData = async ({
             [`${CASE_SAVED_OBJECT}.attributes.owner`]: owner,
           },
         },
-        aggs: getCountsAggregationQuery(CASE_SAVED_OBJECT),
+        aggs: {
+          ...getCountsAggregationQuery(CASE_SAVED_OBJECT),
+          totalUniqueAssignees: {
+            cardinality: {
+              field: `${CASE_SAVED_OBJECT}.attributes.assignees.uid`,
+            },
+          },
+          assigneeFilters: {
+            filters: {
+              filters: {
+                zero: {
+                  bool: {
+                    must_not: {
+                      exists: {
+                        field: `${CASE_SAVED_OBJECT}.attributes.assignees.uid`,
+                      },
+                    },
+                  },
+                },
+                atLeastOne: {
+                  bool: {
+                    filter: {
+                      exists: {
+                        field: `${CASE_SAVED_OBJECT}.attributes.assignees.uid`,
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
       },
     }),
     {}
   );
 
-  const casesRes = await savedObjectsClient.find<
-    unknown,
-    Record<typeof owners[number], { counts: Buckets }> & {
-      counts: Buckets;
-      syncAlerts: Buckets;
-      status: Buckets;
-      users: Cardinality;
-      tags: Cardinality;
-    }
-  >({
+  const casesRes = await savedObjectsClient.find<unknown, CaseAggregationResult>({
     page: 0,
     perPage: 0,
     type: CASE_SAVED_OBJECT,
     aggs: {
       ...byOwnerAggregationQuery,
       ...getCountsAggregationQuery(CASE_SAVED_OBJECT),
+      totalUniqueAssignees: {
+        cardinality: {
```

Comment:
Does it make sense to include `totalUnique, totalWithAtLeastOne, totalWithZero` for all cases?